bucket/simple-dnscrypt.json

```diff
@@ -0,0 +1,68 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2017": "extras/vcredist2017"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt64.msi",
+            "hash": "5BA0C94774ACD7ADE18446B4351E7D28A948C490AEFC50DC6C08C13551F19F7F",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt64.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt64.msi\" | Out-Null}"
+            ]
```

Comment:
I don't understand why this `post_install` section is needed.
Is the MSI file not being removed after extraction?

Reply:
Yes the **.msi** is not being removed after installation. Which is why I have that `post_install` script to remove it.

Reply:
Weird...This should not be happening.

To reassure, I tried installing the manifest (without `post_install`) on my own PC and a VM, and both of them didn't have the problem.

Reply:
Oh maybe I was just being paranoid again. You can remove it from the manifest if you want to.

Reply:
> If your not having any problems, then maybe I was just being paranoid again.

Okay. I will remove it from the manifest.

Probably something like antivirus or indexing services (such as [Everything](https://github.com/ScoopInstaller/Extras/blob/master/bucket/everything.json)) is blocking the file.

Reply:
Probably could be the **Everything.exe** service I run on my computer.